CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.0
+
+## Bug Fixes
+
+* Checkbox no longer overlays the end of the Help field text when the Reverse box is ticked
```

Comment:
Minor - replace 'when the Reverse box is ticked' with 'when the `reverse` prop is set to `true`.